vulntotal/datasources/snyk.py

```diff
@@ -142,7 +192,7 @@ def parse_html_advisory(advisory_html, snyk_id, affected) -> VendorData:
         if "Upgrade" in fixed:
             lower = fixed.index("version") if "version" in fixed else fixed.index("versions")
             upper = fixed.index("or")
-            fixed_versions = "".join(fixed[lower + 1 : upper]).split(",")
+            fixed_versions = [ver.strip(",") for ver in fixed[lower + 1 : upper]]
```

Comment:
Revert this change, this will not reproduce the same result as the original code.
```suggestion
            fixed_versions = "".join(fixed[lower + 1 : upper]).split(",")
```